ONSdigital/rm-action-service
# Fix address used in action request

## Do not merge yet: Need to ensure all related PRs are tested in Pre-prod and discussed with FWM Team first

# Motivation and Context

We are not using the most recent address when a print file is being generated for social reminders. We therefore need to ensure the most recent address is used in action requests, and also to allow business action requests to still process correctly as a result of these changes.

# What has changed



* Move sampleunitref from ActionAddress to ActionRequest.
* Update decorators to reflect these changes.

Related PRs:
[rm-actionsvc-api](https://github.com/ONSdigital/rm-actionsvc-api/pull/34)
[rm-actionexporter-service](https://github.com/ONSdigital/rm-actionexporter-service/pull/66)

# How to test?



Ensure unit and integration tests pass.

The travis build is expected to fail until the actionsvc-api changes are merged.

# Links



[Trello](https://trello.com/c/BNWyG8P1/260-bug200-addresses-are-not-being-updated-when-a-new-one-for-that-ru-is-being-updated-20)
